src/DotVVM.Framework.StartupPerfTests/FileSystemHelper.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.IO;
+
+namespace DotVVM.Framework.StartupPerfTests
+{
+    static internal class FileSystemHelper
+    {
+        public static string CreateTempDir()
+        {
+            var tempDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+            Directory.CreateDirectory(tempDir);
+            return tempDir;
+        }
+
+        public static bool RemoveDir(string dir)
+        {
+            try
+            {
+                if (Directory.Exists(dir))
+                {
+                    Directory.Delete(dir, true);
+                }
+                return true;
+            }
+            catch (Exception)
```

Comment:
The catch was here actually for `IOException` and `UnauthorizedAccessException` - I am using it only to delete a temp folder and in general, it's not a big deal if it fails. 
I've added a retry logic - sometimes it failed to delete it because some files were still locked.